mods/ra/mod.yaml

```diff
@@ -26,7 +27,6 @@ Packages:
 	~snow.mix
 	~interior.mix
 	~scores.mix
-	~scores-counterstrike.mix
```

Comment:
This probably requires a migration rule as well. https://github.com/OpenRA/OpenRA/pull/11528